adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -163,8 +163,10 @@
                     kc.pkceMethod = initOptions.pkceMethod;
                 }
 
-                if (typeof initOptions.disableLogging === 'boolean') {
-                    kc.disableLogging = initOptions.disableLogging;
+                if (typeof initOptions.enableLogging === 'boolean') {
+                    kc.enableLogging = initOptions.enableLogging;
+                } else {
+                    ks.enableLogging = false;
```

Comment:
@jonkoops @mhajas I believe this is the reason why tests are failing on Travis. We need to replace `ks` by `kc`.

Reply:
Ah, nice. I was already going through the logs trying to find the problem. Should be fixed now.